requirements.txt

```diff
@@ -1,266 +1,259 @@
+comm==0.1.2; python_version >= "3.8"
 commonmark==0.9.1; python_full_version >= "3.6.3" and python_full_version < "4.0.0" and (python_version >= "3.7" and python_full_version < "3.9.7" or python_full_version > "3.9.7")
 convertdate==2.4.0; python_version >= "3.7" and python_version < "4"
-cryptography==38.0.2; python_version >= "3.6"
-cssselect==1.1.0; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.4.0"
-cvxpy==1.2.1; python_version >= "3.7"
+cryptography==39.0.0; python_version >= "3.6"
+cssselect==1.2.0; python_version >= "3.8" and python_version < "4.0"
 cycler==0.11.0; python_version >= "3.7"
 cython==0.29.32; python_version >= "3.8" and python_full_version < "3.0.0" and sys_platform == "darwin" or python_full_version >= "3.3.0" and sys_platform == "darwin" and python_version >= "3.8"
-dateparser==1.1.2; python_version >= "3.5"
-datetime==4.7; python_version >= "3.5"
-debugpy==1.6.3; python_version >= "3.7"
+dateparser==1.1.5; python_version >= "3.7"
+datetime==4.9; python_version >= "3.5"
+debugpy==1.6.4; python_version >= "3.8"
 decorator==5.1.1; python_version >= "3.8" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.8" and python_full_version < "3.9.7" or python_full_version > "3.9.7" and python_version >= "3.8"
 defusedxml==0.7.1; python_version >= "3.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.7"
 degiro-connector==2.0.21; python_full_version >= "3.7.1" and python_full_version < "4.0.0"
 deprecation==2.1.0
 detecta==0.0.5; python_version >= "3.6"
 dnspython==2.2.1; python_version >= "3.6" and python_version < "4.0"
-ecos==2.0.10; python_version >= "3.7"
-entrypoints==0.4; python_version >= "3.7" and python_full_version < "3.9.7" or python_full_version > "3.9.7" and python_version >= "3.7"
+entrypoints==0.4; python_version >= "3.8" and python_full_version < "3.9.7" or python_full_version > "3.9.7" and python_version >= "3.8"
 et-xmlfile==1.1.0; python_version >= "3.6"
-exchange-calendars==4.2.3; python_version >= "3.8" and python_version < "4.0" and python_full_version >= "3.7.0"
-executing==1.1.1; python_version >= "3.8"
+exchange-calendars==4.2.4; python_version >= "3.8" and python_version < "4.0" and python_full_version >= "3.7.0"
+executing==1.2.0; python_version >= "3.8"
 fastjsonschema==2.16.2; python_full_version >= "3.7.0" and python_version >= "3.7"
 feedparser==6.0.10; python_version >= "3.6"
 ffn==0.3.6; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.6.0"
 financedatabase==1.0.2
 finnhub-python==2.4.15
-finviz==1.4.4
+finviz==1.4.5; python_version >= "3.8" and python_version < "4.0"
```

Comment:
liar :P

Reply:
That's all poetry! I swearrrrrr

Reply:
been there.  

what poetry functions you run?

Reply:
Just changed tml version to darts=0.23.0

Then poetry update. 

Anything else i should have done?

Reply:
So I broke this recently by doing `poetry update`. I think you just want to do poetry update on darts, or else it resolves the entire chain and updates everything.  So just `poetry update darts` (or add the right version of darts to poetry)